app/services/check_answers/partner_section.rb

```diff
@@ -0,0 +1,28 @@
+module CheckAnswers
```

Comment:
I don' t think you actually need to add this model. The "vanilla" logic for Check Answers will ensure that the fields will only get displayed if the `partner_details` screen is part of the flow, and the `disputed_asset?` logic doesn't apply.

Basically we only need these section models for Check Answers if there's SMOD logic or if there's complex logic around when to display particular fields within a section.